content/vaar-strategi/_index.en.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Get to know our strategy!
+ingress: The public sector must become a world leader in digitization - and The Norwegian Digitalisation Agency (Digdir) must be the government's foremost tool for faster and more coordinated digitization of society. Our efforts - in close collaboration with others - will lead to citizens, business and voluntary organizations experiencing one digital public sector and a simpler digital everyday life. That is Digdir's ambition and vision.
+
+#SEO
+description:
+
+external_url: http://www.digdir.no/digdir/digitaliseringsdirektoratets-strategi/2497
```

Comment:
Confusing when these are not opened in new tab?

Reply:
Agree! 👍 Fixed!